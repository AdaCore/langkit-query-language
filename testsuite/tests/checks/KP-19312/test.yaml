driver: checker
rule_name: KP_19312
project: prj.gpr
