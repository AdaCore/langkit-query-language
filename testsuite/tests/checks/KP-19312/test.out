main.adb:35:44: rule violation: possible occurrence of KP 19312
35 |    S_1 : constant Integer_Sets.Set := [for E of I when False => E];  --  FLAG
   |                                            ^^^^^^^^^^^^^^^^^

main.adb:51:8: rule violation: possible occurrence of KP 19312
51 |    for Elem in I when False loop        --  FLAG
   |        ^^^^^^^^^^^^^^^^^^^^

main.adb:55:8: rule violation: possible occurrence of KP 19312
55 |    for Elem of I when False loop        --  FLAG
   |        ^^^^^^^^^^^^^^^^^^^^

